src/MatLang.jl

```diff
@@ -1,23 +1,215 @@
 module MatLang
-using Requires
 
-export clcM
+using LinearAlgebra
+
+using Requires # for clcM
+import BlockDiagonals # for blkdiagM
+
+# Main Functions
+export clcM, zerosM, onesM, randM, eyeM, diagM, blkdiagM
+
+# Julia functions with M suffix
+# export
+
+################################################################
+################################################################
+# Command Window (REPL console):
+################################################################
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
 end
-################################################################
-
-
-################################################################
 # Conditional REQUIRE - https://github.com/MikeInnes/Requires.jl
 function __init__()
     @require Juno="e5e0dc1b-0480-54bc-9374-aad01c23163d" clcM() = Juno.clearconsole();
 end
+################################################################
+
+
+################################################################
+################################################################
+# Matrices and Arrays:
+################################################################
+################################################################
+"""
+    zerosM(idx)         # square idx*idx matrix
+    zerosM(Type,idx)    # square idx*idx matrix
+    zerosM(sizeAsArray) # non-efficient Matlab way
+    zerosM(Type, sizeAsArray) # non-efficient Matlab way
+
+# Example
+mZeros0=zerosM(2); # same as zeros(2,2)
```

Comment:
you need to quote these examples with triple quotes.

Reply:
My bad I will fix this.